VIC_RBM/model_run/result_analysis/scripts/plot_flow_future.py

```diff
@@ -0,0 +1,129 @@
+s_hist_to_plot = my_functions.select_time_range(s_hist, plot_hist_start_date, plot_hist_end_date)
+list_s_future_to_plot = []
+for i in range(nfuture):
+	s_future_to_plot = my_functions.select_time_range(list_s_future[i], plot_future_start_date, plot_future_end_date)
+	list_s_future_to_plot.append(s_future_to_plot)
+
+#========================================================
+# plot
+#========================================================
+model_info = 'VIC runoff, historical -  1/8 deg, Maurer forcing, Andy Wood setup\n          VIC runoff, climate change scenarios - 1/8 deg, Reclamation archive (GCM forcing)\n          Route: Lohmann, Wu flowdir, Andy Wood setup'
+
+##============== plot daily data ===============#
+#fig = my_functions.plot_time_series(plot_date=True, \
+#		list_s_data=[s_hist_to_plot]+list_s_future_to_plot, \
+#		list_style=[hist_style]+list_style_future, \
+#		list_label=['historical']+future_route_names, \
+#		plot_start=plot_hist_start_date, plot_end=plot_future_end_date, \
+#		ylabel='Flow (cfs)', title='%s, %s' %(usgs_site_name, usgs_site_code), 
+#		fontsize=16, legend_loc='upper left', \
+#		time_locator=time_locator,\
+#		add_info_text=True, model_info=model_info, stats='daily, no stats')
+#
+#fig.savefig('%s.future.flow.daily.png' %output_plot_basename, format='png')
+#
+##============== plot monthly data ===============#
+#fig = my_functions.plot_monthly_data(list_s_data=[s_hist_to_plot]+list_s_future_to_plot, \
+#		list_style=[hist_style]+list_style_future, \
+#		list_label=['historical']+future_route_names, \
+#		plot_start=plot_hist_start_date, plot_end=plot_future_end_date, \
+#		ylabel='Flow (cfs)', title='Monthly mean, %s, %s' %(usgs_site_name, usgs_site_code), 
+#		fontsize=16, legend_loc='upper left', \
+#		time_locator=time_locator,\
+#		add_info_text=True, model_info=model_info, stats='Monthly mean')
+#
+#fig.savefig('%s.future.flow.monthly.png' %output_plot_basename, format='png')
+
+#============== plot seasonality data ===============#
+fig = my_functions.plot_seasonality_data(list_s_data=[s_hist_to_plot]+list_s_future_to_plot, \
+		list_style=[hist_style]+list_style_future, \
+		list_label=['historical']+future_route_names, \
+		plot_start=1, plot_end=12, \
+		ylabel='Flow (cfs)', \
+		title='Monthly mean seasinality, %s, %s' %(usgs_site_name, usgs_site_code), 
+		fontsize=16, legend_loc=None, \
+		xtick_location=range(1,13), \
+		xtick_labels=['Jan','Feb','Mar','Apr','May','Jun','Jul','Aug','Nov','Oct','Nov','Dec'], \
+		add_info_text=True, model_info=model_info, stats='Monthly mean; historical - WY%d-%d; climate change scenario - WY%d-%d' %(plot_hist_start_date.year+1, plot_hist_end_date.year, plot_future_start_date.year+1, plot_future_end_date.year))
+
+fig = plt.savefig('%s.future.flow.seas.png' %output_plot_basename, format='png')
+
+#============== plot annual mean flow ===============#
+fig = my_functions.plot_WY_mean_data(list_s_data=[s_hist_to_plot]+list_s_future_to_plot, \
```

Comment:
your plot_WY_mean_data appears twice in the my_functions.py script! This could pose problems if you make developments to the first that will get overwritten by the second!